packages/server/src/service/mg.ts

```diff
@@ -92,6 +94,50 @@ export const sendSignupEmail = async (
   return mg.messages().send(emailData)
 }
 
+
+export const sendCSVEmail = async (
+  csv: string,
+  uid: string,
+  name: string | undefined,
+  emails: string[],
+  oid: string,
+) => {
+  try {
+    const attachment = [new mg.Attachment({
+      data: Buffer.from(csv),
+      contentType: 'text/csv',
+      filename: 'saved_sign_ups.csv',
+    })]
+    const mgData = {
+      from: processEnvOrThrow('MG_FROM_ADDR'),
+      'h:Reply-To': [processEnvOrThrow('MG_REPLY_TO_ADDR'), ...emails].join(','),
+      'v:metadata': { oid },
+    }
+    const email: mailgun.messages.SendData = {
+      to: emails,
```

Comment:
Can we tell which user specifically requested the download and just email it to them? This could just be fixed in the future (open a GitHub issue though for a code clean up).

Reply:
We're only emailing the user who has requested the download, but since you can have more than one email attached to your google account we email to all of their emails.

Reply:
Oh I didn't know that you could have multiple emails attached to a Google account.